js/main.js

```diff
@@ -166,15 +259,85 @@ function removeFromBasket(e) {
   } else {
     BASKET = BASKET.filter((el) => el.id !== productInBasket.id);
   }
-  
+
+  saveBasketInLocalStorage();
   renderBasket();
 }
 
+// Sorting products
+function sortCurrentList() {
+  const sorterElement = document.querySelector(".sort__array");
+  const sorterValue = sorterElement.options[sorterElement.selectedIndex].value;
+  switch (sorterValue) {
+    case "z-a":
+      CURRENT_PRODUCTS_LIST.sort((x, y) => y.title.localeCompare(x.title));
+      renderProducts();
```

Comment:
renderProducts moze byc poza switchem, i tak to jest zawsze ostatnia akcja

Reply:
Zmienione